dev/md/options.md

```diff
@@ -10,7 +10,7 @@
 | `mobileBreakpoint`     | number                 | `768`             | Activate mobile layout when the screen size is <=768px                                                                   |
 | `closeButtonInnerText` | string                 | `svg/icon`        | Custom `innerText` of the close button                                                                                   |
 | `triggerSelector`      | CSS selector           | `""`              | Click on this trigger will also toggle the popup                                                                         |
-| `outerClass`           | string                 | `""`              | Extra class to popup outer `.easy-popup`                                                                                 |
+| `outerClass`           | string                 | `""`              | Extra classes to popup outer `.easy-popup`                                                                               |
```

Comment:
Add this sentence:

Use white space for multiple classes, e.g. `"class1 class2"`.

Reply:
I updated. Can you check it again? Thanks.